### ISSUE TITLE
The github wiki section needs admins to provide official documentation to CTRLR

### ISSUE THREAD

**Issue by damiensellier** [RomanKubiak/ctrlr#329]
Hi,
The title says it all, that would be awesome to provide the community a well formated wiki section on this official github.
Thanks

**Comment by dobo365** [comment]
I support this and would agree to administrate it and populate it together with 1 or 2 others.
Don't think we should be many but 2-3 seems OK.
Of course with input coming from many ;-)

**Comment by RomanKubiak** [comment]
yeah just let me know who has the time and will to do that, i will add them to the project

**Comment by damiensellier** [comment]
you can count me in Roman. I'm pretty sure John @dnaldoog would like to be involved in this as well. Just need his confirmation. @Godlike-Productions  is also currently active on the threads with some others. Let's check who's up for that and that will be awesome ;)

**Comment by dobo365** [comment]
> yeah just let me know who has the time and will to do that, i will add them to the project

Great, Roman! You can count on me (goodweather - dobo365). Thx a lot...Dominique
Then I can copy many things from my Step by step guide word doc into this wiki
I have now divided it in 3 parts
- Building a panel step by step
- Automating your panels with Lua programming (explanation of Lua, basic Ctrlr methods, advanced Ctrlr methods)
- Reference guide (this one is the least advanced part)

**Comment by dnaldoog** [comment]
Yes, I would like to be included as an Admin if possible. :)

**Comment by DnaldeT** [comment]
Needed a little persuasion. But you can count me in as well if needed.

**Comment by Godlike-Productions** [comment]
I'm happy to be an admin as well.  I've had a lot of experience with wiki's in the past.  We really need to make sure that editing is not available publically or we are going to get smashed with spammers.
